docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -150,16 +152,18 @@ image
 
 ## Monitoring active adapters
```

Comment:
@sayakpaul Could you please check this section.

Reply:
Code checks out here so I'm going to merge 😄